glossary/index.md

```diff
@@ -216,6 +226,12 @@ A [peer](/glossary/#peer) [node](/glossary/#node) that is currently connected to
 
 A participant is able to sync state, and broadcast and receive transactions, and potentially be a [validator](/glossary/#validator) node.
 
+## participation rate
+
+The percentage of "circulating" [newton](/glossary/#newton) supply that is bonded to registered [validator](/glossary/#validator) nodes on an [Autonity network](/glossary/#autonity-network).
+
```

Comment:
The participation rate is the percentage of circulating NTNs that are staked in the CC (bonded to CC validators), as the NTNs that are bonded to paused or inactive nodes should be excluded (see notations https://github.com/clearmatics/autonity-protocol/discussions/213#discussioncomment-7117494)